.theia/settings.json

```diff
@@ -0,0 +1,3 @@
+{
```

Comment:
I think you can add these settings.json files to your .gitignore file, they don't need to be in your repo. But they aren't hurting anything, so it's up to you!

Reply:
So, do I just put .theia in the .gitignore file? Or do I put .theia/settings.json?

Reply:
Oh also should I put the .vscode stuff in the .gitignore as well?

Reply:
Yeah I think that sounds right to me!